src/strategies/greenblockcapital-stake/README.md

```diff
@@ -0,0 +1,22 @@
+# GreenBlock Capital stake strategy
+
+Allows the tokens staked in chef conttracts to be used to calculate voter scores.
+
+## Examples
+
+Here is an example of parameters:
+
+```json
+{
+  "strategies": [
+    [
+      "greenblockcapital-stake",
+      {
+        "poolAddress": "0xe322ea8cbc9d0a5b12110c64f07f489e8bfe1788",
+        "symbol": "GBT",
+        "decimals": 18
+      }
+    ]
+  ]
+}
```

Comment:
```suggestion
{
        "address": "0xe322ea8cbc9d0a5b12110c64f07f489e8bfe1788",
        "symbol": "GBT",
        "decimals": 18
}
```

Reply:
You're right, it has to work that way because that's how it's configured.  I forgot to update it in the README file